_data/pixels.json

```
@@ -1262,4 +1263,4 @@
     {"y": 59, "x": 58, "color": "#108DE0", "username": "polivierstpch"},
     {"y": 59, "x": 59, "color": "#000000", "username": "curtca"}
   ]
-}
+}
```

```suggestion
}

```

Try this @MishManners

---

Oh thank you YG Dev!